README.md

```diff
@@ -14,10 +14,11 @@ Running Ting
 3. Make a copy of `config/common.json` into `config/local.json` and add your
    settings.
 4. Install the dependencies required to run Django requirements installation
-   by running `sudo apt-get install python-dev libmysqlclient-dev libffi-dev` and 
-   `pip install MySQL-python`
+   by running `sudo apt-get install python-dev libmysqlclient-dev libffi-dev`,
+   `sudo apt-get install python-pip` and `pip install MySQL-python`.
 5. **Optional**. You can skip to step 6 although it's highly recommended that
    you follow these instructions.
+   Install `virtualenv` by running `sudo pip install virtualenv`.
```

Comment:
I think it shouldn't be merged. Step 5 talks explicitily for virtual enviroment and is marked as optional. Thus, the command should stay as is and not to be merged.